### ISSUE TITLE
Properly destroying Drift

### ISSUE THREAD

**Issue by tarponjargon** [imgix/drift#60]
I'm trying to get Drift to work with Ember and am having some success.   Since there aren't any page reloads I want to be sure that I am cleaning up all remnants of Drift on transition out of the route that uses it.

Right now I'm doing this:

```
let drift = New Drift(...);
...
destroyMethod() {
    drift.destroy();
    window.Drift = null;
    drift = null;
}
```

And that seems to work, but is there anything else I need to address?  Perhaps some DOM elements that could potentially be left hanging around?

Thanks for this plugin - I love it!

**Comment by michaelbromley** [comment]
I'm using Drift in an Angular application. In the "ngOnDestroy" lifecycle hook, I call the Drift `destroy()` method, but if the drift bounding box is currently being displayed, it will not get cleaned up.

Stepping through the `destroy()` call, I see that it only removes event listeners, and does not check to see if any of its created DOM elements are still attached.

See this illustration of the problem, wherein I am using the "back" button on my mouse to navigate away:

![2019-05-21_09-16-27](https://user-images.githubusercontent.com/6275952/58075973-4e02b300-7ba9-11e9-845b-d390136b7940.gif)

**Comment by sherwinski** [comment]
Hey @michaelbromley, thanks for the workaround and I definitely agree with your suggestion. I plan to work on incorporating this fix sometime into the project in the near future 👍 

Relates to #73

**Comment by sherwinski** [comment]
Hi everyone, the fix for this is released now in [v1.3.5](https://github.com/imgix/drift/releases/tag/v1.3.5) 🎉
